NanaZip.Modern/MainWindowToolBarPage.cpp

```diff
@@ -155,13 +155,12 @@ namespace winrt::NanaZip::Modern::implementation
         this->m_DispatcherQueue =
             winrt::DispatcherQueue::GetForCurrentThread();
 
-        this->m_StoreContext = winrt::StoreContext::GetDefault();
-        if (this->m_StoreContext)
-        {
-            winrt::check_hresult(
-                this->m_StoreContext.as<IInitializeWithWindow>()->Initialize(
-                    this->m_WindowHandle));
-        }
+        std::wstring sponsorButtonLabel = L"[";
+        sponsorButtonLabel += Mile::WinRT::GetLocalizedString(
```

Comment:
According to the definition of Mile::WinRT::GetLocalizedString.

https://github.com/ProjectMile/Mile.Windows.Helpers/blob/49dd11f06cd3e35e530ce32304dd687bf58c4c8d/Mile.Helpers/Mile.Helpers.CppWinRT.h#L103-L111

I think we should add the fallback string. It will better than showing "[NanaZip.Modern/MainWindowToolBarPage/SponsorButton/AcquireText]" when failed to get the language resource.

Kenji Mouri

Reply:
This is done in 074cd62